modules/tensor_mechanics/src/materials/abaqus/AbaqusUMATStress.C

```diff
@@ -302,8 +317,9 @@ AbaqusUMATStress::computeQpStress()
   _stress[_qp] = RankTwoTensor(
       _aqSTRESS[0], _aqSTRESS[1], _aqSTRESS[2], _aqSTRESS[5], _aqSTRESS[4], _aqSTRESS[3]);
 
-  // Rotate the stress state to the current configuration
-  _stress[_qp].rotate(_rotation_increment[_qp]);
+  // Rotate the stress state to the current configuration, unless HughesWinget
+  if (_decomposition_method != "HughesWinget")
```

Comment:
```suggestion
  if (_decomposition_method != ComputeFiniteStrain::DecompMethod::HughesWinget)
```

we should not have string comparisons in the QP loop